lib/ui/home/preference/preference.dart

```diff
@@ -1,6 +1,6 @@
 import 'dart:io';
 import 'dart:ui';
-import 'dart:html' as html;
```

Comment:
Just want to double check we are commenting out import dart as html, thank you!

Reply:
I had an error with that on my computer that prevented me to "flutter run", I think my computer didn't have enough required files for that, however when I commented that out, I could run the project.